testing/values.go

```diff
@@ -28,10 +28,11 @@ const (
 	FirstConnectionID = "connection-0"
 
 	// Default params constants used to create a TM client
-	TrustingPeriod     time.Duration = time.Hour * 24 * 7 * 2
-	UnbondingPeriod    time.Duration = time.Hour * 24 * 7 * 3
-	MaxClockDrift      time.Duration = time.Second * 10
-	DefaultDelayPeriod uint64        = 0
+	TrustingPeriod               time.Duration = time.Hour * 24 * 7 * 2
+	UnbondingPeriod              time.Duration = time.Hour * 24 * 7 * 3
+	MaxClockDrift                time.Duration = time.Second * 10
+	DefaultDelayPeriod           uint64        = 0
+	DefaultTimeoutTimestampDelta               = 1 << 12
```

Comment:
what's the reason for using the `<<` operator here? Can't we just use something like `time.Minute` ?

Reply:
No special reason, just wanted to have a sufficiently large number. I changed it now.